src/expandableCalendar/week.tsx

```diff
@@ -12,7 +12,8 @@ import Day from '../calendar/day/index';
 // import BasicDay from '../calendar/day/basic';
 
 
-interface Props extends CalendarProps {
+// TODO: current type should be a string
+interface Props extends Omit<CalendarProps, 'current'> {
```

Comment:
Why no do the same for ExpandableCalendar until we start the migration from XDate to string?